clients/src/test/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImplTest.java

```diff
@@ -2489,7 +2498,11 @@ private void testRevocationCompleted(MembershipManagerImpl membershipManager,
         verify(subscriptionState).markPendingRevocation(anySet());
         List<TopicPartition> expectedTopicPartitionAssignment =
                 buildTopicPartitions(expectedCurrentAssignment);
-        verify(subscriptionState).assignFromSubscribed(new HashSet<>(expectedTopicPartitionAssignment));
+        verify(subscriptionState).assignFromSubscribedAwaitingCallback(any(), eq(new HashSet<>()));
```

Comment:
here we probably know what we expect as assignment in the first param instead of any() (is this maybe a mixup with the if/else verify right below?